framework/validators/UniqueValidator.php

```diff
@@ -234,7 +244,20 @@ private function prepareConditions($targetAttribute, $model, $attribute)
             $conditions = [$targetAttribute => $model->$attribute];
         }
 
-        return $conditions;
+        if (!$model instanceof ActiveRecord) {
+            return $conditions;
+        }
+
+        // Add table prefix for column.
+        $targetClass = $this->getTargetClass($model);
+        $tableName = $targetClass::tableName();
+        $conditionsWithTableName = [];
+        foreach ($conditions as $columnName => $columnValue) {
```

Comment:
Maybe use just `$column => $value` instead? These are loop's internal variables, also I don't see any coincidence with other similar variables or anything that can confuse understanding of a shorter variation.